spec/shared_context.rb

```diff
@@ -109,6 +119,25 @@ def integration_context
   $config_synergy ||= OneviewSDK::Config.load(@config_path_synergy)
 end
 
+# Must set the following environment variables:
+#   ENV['ONEVIEWSDK_INTEGRATION_I3S_CONFIG'] = '/full/path/to/one_view/config.json'
+# Or use the default paths:
+#   spec/integration/one_view_i3s_config.json
+def integration_context_i3s
+  default_config = 'spec/integration/one_view_i3s_config.json'
+  @config_path_i3s ||= ENV['ONEVIEWSDK_INTEGRATION_I3S_CONFIG'] || default_config
```

Comment:
Do we reaaaally need to mention ONEVIEW in the Env? I mean, it's related to OneView but it isn't OneView...